lib/Tool/Admin.php

```diff
@@ -103,7 +104,15 @@ public static function getMinimizedScriptPath($scriptContent)
             '_dc' => \Pimcore\Version::getRevision()
         ];
 
-        return '/admin/misc/script-proxy?' . array_toquerystring($params);
+        if ($asUrl) {
+            @trigger_error(
+                'Calling Pimcore\Tool::getMinimizedScriptPath with $asUrl true is deprecated and will be removed with Pimcore 7.1.0',
```

Comment:
Why remove it with Pimcore `v7.1` and not with `v7.0`?

Reply:
good catch, can be removed with 7.0 as well

Reply:
#6513